### PR TITLE
GetAll for EarnedAchievements endpoint

### PR DESCRIPTION
**Pull request by Anarios**

### CODE REVIEW

File: BadgeUpClient/Http/HttpQuery.cs [hunk 1]
```diff
@@ -22,5 +25,27 @@ public override string ToString()
 
 			return null;
 		}
+
+
+		public static string GetQueryStringFromObject(object param)
```
Comment (MarkHerhold) [inline BadgeUpClient/Http/HttpQuery.cs]: Wouldn't it make more sense to move this to an interface that all of the *QueryParams classes extend so we can override individual implementations if needed?

Comment (Anarios) [reply BadgeUpClient/Http/HttpQuery.cs]: This one is generic enough to convert any object to a query string, so I thought that HttpQuery is an OK place for it. Can be moved to base class for QueryParams if you prefer.

Comment (MarkHerhold) [reply BadgeUpClient/Http/HttpQuery.cs]: Let's move it so we don't have issues where someone tries to serialize some random object

**Comment by MarkHerhold** [comment]
Thank you @Anarios 👍

**Pull request merged**